inception-doc/src/main/resources/META-INF/asciidoc/user-guide/formats-tei.adoc

```diff
@@ -28,7 +28,7 @@ For more details about the supported element types, see the link:https://dkpro.g
 | link:https://www.tei-c.org/release/doc/tei-p5-doc/en/html/index.html[TEI P5]
 | yes
 | yes
-| yes
-| UIMA XMI CAS
+| no
+| Lemma, POS (xpos), named entities
```

Comment:
I think for named entities only the "value" feature is supported - not the identifier feature. Since you mention xpos for the POS layer, maybe that should be mentioned here as well.

Reply:
you are right, changed it